### PR TITLE
Enchancement/59/Update stats command to include more information about the bot.

### PR DESCRIPTION
**Pull request by JoshQuaintance**
Added Uptime on the stats command, showing the uptime for the bot.
Added Online Time on the stats command, showing what time the bot started.
Changed the command calling to pass in the Client variable to get Uptime.

### CODE REVIEW

File: src/commands/stats.js [hunk 1]
```diff
@@ -6,8 +6,44 @@ module.exports = {
...
+          ? `0${Math.floor(totalSeconds % 60)} seconds`
+          : `${Math.floor(totalSeconds % 60)} seconds`;
+
+      const date = new Date();
+      const hour =
+        date.getHours() < 10 ? `0${date.getHours()}` : date.getHours();
+      const mins =
+        date.getMinutes() < 10 ? `0${date.getMinutes()}` : date.getMinutes();
+      const tz = /\((.*)\)/.exec(new Date().toString())[1];
+      const am_pm = hour >= 12 ? 'PM' : 'AM';
```
Comment (bradtaniguchi) [inline src/commands/stats.js]: Move the 2 distinct code blocks to their own functions that return the strings provided to the `statsEmbed` block.

**Review by bradtaniguchi — COMMENTED** [review]
How do you feel about the custom date formatting you have? 

I'm the type of developer where I see a date calculation problem and walk straight to a lib solution (lately its been [luxon](https://moment.github.io/luxon/)) 

If your even slightly unsure about the code you have right now we can go use something like luxon instead and not worry about this code, nor worry about maintaining it if there are bugs/improvements down the line.

File: src/commands/stats.js (hunk 1 above)
Comment (JoshQuaintance) [reply src/commands/stats.js]: I'll do this tomorrow. But I'll look into luxon and maybe even use it.

Comment (JoshQuaintance) [reply src/commands/stats.js]: Moved code blocks into each corresponding function that will return string that contains each corresponding data.

**Comment by JoshQuaintance** [comment]
The `getWakeTime()` function doesn't work to it's full potential yet, it needs a JSON file to store it's data. I don't know if you want me to add a JSON file or something else, just clarifying.

**Comment by bradtaniguchi** [comment]
> I'll do this tomorrow. But I'll look into luxon and maybe even use it.

I'll leave this PR open until you review Luxon, which could be used for both situations.

I'd also recommend renaming the two as `uptime` and `wakeTime` sound too similar, but this can be done later if you do use Luxon. 

> The getWakeTime() function doesn't work to it's full potential yet, it needs a JSON file to store it's data. I don't know if you want me to add a JSON file or something else, just clarifying.

If we need to have a JSON file with a date to represent "last deployed", then just return any date from this function, as it wont need to do any calculations, it will just show the date saved in the JSON file the bot is deployed with. (this can be done in another PR where we actually have the JSON file)

**Comment by JoshQuaintance** [comment]
Made the function into it's own file so it can be called in the main file. On the bot start, it will generate and return the date, then it can be pushed into the JSON file.

**Review by bradtaniguchi — APPROVED** [review]
LGTM, the `botOnlineAt` variable might be saved differently in the future but should be fine enough for now.

**Pull request merged**